src/thunderbots/software/ai/primitive/directwheels_primitive.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include "ai/primitive/primitive.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+
+class DirectWheelsPrimitive : public Primitive
+{
+   public:
+    static const std::string PRIMITIVE_NAME;
+    // Power is a fraction of the total power we can apply to the robots,
+    // with +-255 being the max/min, and 0 being no power.
+    /**
+     * Creates a new Move Primitive
+     *
+     * @param robot_id the id of the robot
+     * @param wheel0_power a value between -255 and 255, where positive is clockwise
+     * @param wheel1_power a value between -255 and 255, where positive is clockwise
+     * @param wheel2_power a value between -255 and 255, where positive is clockwise
+     * @param wheel3_power a value between -255 and 255, where positive is clockwise
```

Comment:
Hmm, is it possible to have some more descriptive names for wheel power? Maybe `left_front_wheel_power`, etc.?

Reply:
`wheel_0` = "front left"
`wheel_1` = "back left"
`wheel_2` = "back right"
`wheel_3` = "front right"